README.md

```diff
@@ -1,2 +1,30 @@
-# scraped_page
+# ScrapedPage
+
 Write declarative scrapers in Ruby
+
+## Usage
+
+Create a subclass of `ScrapedPage` for each page you wish to scrape.
+
+```ruby
+require 'scraped_page'
+
+class ExamplePage < ScrapedPage
+  field :title do
+    noko.at_css('h1').text
+  end
+  
+  field :more_information do
+    noko.at_css('a')[:href]
+  end
+end
+
+puts ExamplePage.new('http://example.com').to_h
+# => { :title => "Example Domain", :more_information => "http://www.iana.org/domains/reserved" }
+```
+
+The default strategy for retrieving pages is to make an http request to retrieve them. If you'd also like to archive a copy of the page you're scraping into a git branch, you can pass a `:strategy` option to the constructor:
+
+```ruby
+ExamplePage.new('http://example.com', strategy: ScrapedPage::LiveRequestArchiveStrategy)
```

Comment:
I suppose if scrapers provide their own strategies then it's best for them to work in the same way as the built in classes.

The reason I've done it this way is because when doing the prototype in the guernsey scraper I [needed to pass the `url` to the `strategy`](https://github.com/everypolitician-scrapers/guernsey-2016/blob/1f76f009d6dc487141b918f618aa3ed0dc663103/lib/scraped_page.rb#L14) and passing in a class seemed like the most straightforward way to do that. This does suggest that I've perhaps designed the interface to suit my needs as an implementor rather than a user though 🤔